amundsen_application/static/js/components/common/ShimmeringTagListLoader/styles.scss

```diff
@@ -0,0 +1,29 @@
+@import 'variables';
+
+$shimmer-loader-tag-height: 36px;
+$shimmer-loader-tag-border-radius: 4px;
```

Comment:
nit: I assume these two values were chosen to match `TagInfo`, so we perhaps we could add some shared tag-info variables in `_variables-default` and use them both here and in `TagInfo` for easier update if we ever change the size.

Reply:
Yeah, makes sense!
I will check those up, thanks!

Reply:
I was only able to reuse the border radius value.